packages/algolia/test/client.test.ts

```diff
@@ -15,7 +17,37 @@ describe('Algolia Search Client', () => {
   const algoliaSearchIndex = getAlgoliaSearchIndexMock();
   const algoliaSearchClient = new AlgoliaSearchClient(algoliaSearchIndex);
 
-  test('Should save the Research Output', async () => {
+  it('Should do batch on entities', async () => {
```

Comment:
`test`

but also - this test already suggests that maybe we dont need a `batch` method, just a `saveMany` method? This seems quite generic in comparison to what we have built for the algolia client.